app/javascript/packages/document-capture/components/hybrid-doc-capture-warning.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext } from 'react';
+import type { ReactNode } from 'react';
+import { useI18n, formatHTML } from '@18f/identity-react-i18n';
+import { getConfigValue } from '@18f/identity-config';
+import { Alert } from '@18f/identity-components';
+import ServiceProviderContext from '../context/service-provider';
+
+function formatWithStrong(text: string): ReactNode {
+  return formatHTML(text, {
+    strong: ({ children }) => <strong>{children}</strong>,
+  });
+}
+
+interface HybridDocCaptureWarningProps {
+  /**
+   * A class string to append to root element
+   */
+  className?: string;
+}
+
+function HybridDocCaptureWarning({ className = '' }: HybridDocCaptureWarningProps): JSX.Element {
+  const { t } = useI18n();
+  // Determine the Service Provider name to display,
+  // in some circumstances.
+  // If there is no SP, we default to the appName
+  const spContext = useContext(ServiceProviderContext);
+  const serviceProviderName = spContext.name;
```

Comment:
Optional: Could destructure this:

```suggestion
  const { name: serviceProviderName} = useContext(ServiceProviderContext);
```